backend/pyproject.toml

```diff
@@ -44,5 +42,7 @@ httpx = "^0.23.3"
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
+ignore = ["E501"]
```

Comment:
We're ignoring the line too long error. We did not enforce this before and it would cause some problems.